docs-2.0/2.quick-start/6.useful-links.md

```diff
@@ -0,0 +1,55 @@
+# Useful links
+
+## API Clients by Nebula Graph
+
+| links | commit id |
+| -- | -- |
+| [C++ Client](https://github.com/vesoft-inc/nebula-cpp/tree/v2.0.0) | 7305c72 |
+| [Go Client](https://github.com/vesoft-inc/nebula-go) | 542ed24 |
+
+The follow repositories of v2.0.0 are not released yet. (TODO Mar. 23 2021):
+* [Python Client](https://github.com/vesoft-inc/nebula-python)
+* [Java Client](https://github.com/vesoft-inc/nebula-java)
+* [Rust Client](https://github.com/vesoft-inc/nebula-rust)
+* [Node.js Client](https://github.com/vesoft-inc/nebula-node)
+* [HTTP Client](https://github.com/vesoft-inc/nebula-http-gateway)
+
+## Graph tools
+
+| links | commit id |
+| -- | -- |
+| [Command Line Console](https://github.com/vesoft-inc/nebula-console/tree/v2.0.0-ga)  |  1f32236 |
+
+The follow repositories of v2.0.0 are not released yet. (TODO Mar. 23 2021):
+* [Studio](https://github.com/vesoft-inc/nebula-docker-compose)
+* [Dashboard](https://github.com/vesoft-inc/nebula-stats-exporter)
```

Comment:
Dashboard is on https://github.com/vesoft-inc-private/nebula-dashboard
The one you have here is for Nebula Stats Exporter